shared/src/test/scala/org/http4s/websocket/WebsocketSpec.scala

```diff
@@ -72,6 +72,24 @@ class WebsocketSpec extends Specification {
       new String(msg.data, UTF_8) should_== "Hello"
     }
 
+    "encode a close message" in {
+      val frame = Close(1000, "some reason").right.get
+      val msg = decode(encode(frame, true), false)
```

Comment:
This would be a great place for a property test.

Reply:
Sure, guess I'd better learn how to do that then :D